### ISSUE TITLE
[feature request] firefox?

### ISSUE THREAD

**Issue by AlJohri** [patheticGeek/improved-potato#1]
Any chance you could port this extension to also work on Firefox?

**Comment by patheticGeek** [comment]
Yeah, need to make a manifest file for it and publish. Will do over the weekend.

**Comment by Blade694** [comment]
Any updates on the firefox port?

**Comment by david-shepard** [comment]
I know it's been a long time but I've added support for this in https://github.com/patheticGeek/improved-potato/pull/2
